frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./NavBar.css"
+import logo from "../assets/images/logo.png"
+
+const NavBar = ({ nav_1, nav_2, nav_3, nav_4, nav_5 }) => {
```

Comment:
If we are passing these to the component, would it make more sense to pass them as an array that we can map over instead of hard coding every list item? It could be an array of objects with a Text property and a Type property to decide whether it's a regular item of a log in/log out item, or alternatively they could be two separate arrays for simplicity.

Reply:
Hi Virginia, I made a new commit with the corrections you asked for :)